Dockerfile

```diff
@@ -3,4 +3,4 @@ ADD build/libs/payments.jar /srv/payments.jar
 WORKDIR /srv
 EXPOSE 8080
 RUN mkdir config
```

Comment:
This is the folder where the `oidc-client-registration.json` will we written. We're not quite sure whether this is important.